prepare_beta.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs');
+const _ = require('lodash');
+
+const packageJson = require('./package.json');
+const aptlyJson = JSON.parse(fs.readFileSync('./.aptly.conf'));
+
+// -------
+
+const BUILD_TYPE_PATH = 'type';
+
+const PRODUCTION_TYPE = undefined;
+const BETA_TYPE = 'beta';
+
+const WINDOWS_GENERIC_PATH = 'build.win.publish[0].url';
+const WINDOWS_S3_PATH = 'build.win.publish[1].path';
+const OSX_GENERIC_PATH = 'build.mac.publish[0].url';
+const OSX_S3_PATH = 'build.mac.publish[1].path';
+
+const PRODUCTION_PATH = 'https://updates.signal.org/desktop';
+const BETA_PATH = 'https://updates.signal.org/desktop-beta'
```

Comment:
Do we need this? Isn't it enough to have '-beta' in the version number?

Reply:
Yeah, I think we can put beta in the version number. Our git history will have very clear `X-beta-1`, `X-beta-2`, `X` progression.